TK.json

```diff
@@ -234,7 +234,7 @@
 			"wowID":"32458"
 		},
 		"Verdant Sphere":{
-			"prio":["MOMMYSPANKMEHARD(1)","Spagoose(1)","Ele Shaman/Hunter/Casters/Healers/Enh","EP/GP"],
+			"prio":["Mazzo(1)","Tatertot(1)","Mommy(1)","(prestige(1)","Ele Shaman/Hunter/Casters/Healers/Enh","EP/GP"],

```

Comment:
Caster neck is bis all phase for mages / ele sham and none of them have it yet; healers have other options in the shrahaz neck.  What do you think about tater - prestige - micro - raehn then mazzo oph? The healers definitely deserve it, but you and oph can take shrahaz neck earlier in t6